opl/investigator/config.py

```diff
@@ -75,6 +75,10 @@ def load_config(conf, fp):
         assert not conf.history_es_server.endswith("/")
         conf.history_es_index = data["history"]["es_index"]
         conf.history_es_query = data["history"]["es_query"]
+        if "es_server_user" in data["history"]:
+            conf.es_server_user = data["history"]["es_server_user"]
+            conf.es_server_pass_env_var = data["history"]["es_server_pass_env_var"]
+            conf.es_server_verify = data["history"]["es_server_verify"]
```

Comment:
If I have removed `If condition` then most of services fail at this line because investigator.conf file is not updated with username, password and verify flag. The plan is to start with front-end service and slowly propagate changes to other services, once it is complete then `If condition` is not longer required.

Reply:
I will keep it for now until all investigator.conf files are updated across the services.